widgetry/src/widgets/checkbox.rs

```diff
@@ -185,7 +185,7 @@ impl Checkbox {
                 bottom: 14.0,
```

Comment:
Maybe not the right place to comment, but
![Screenshot from 2021-01-21 11-25-30](https://user-images.githubusercontent.com/1664407/105401765-637c7b80-5bdb-11eb-893a-4b6f416d54d5.png)
The toggle style now looks kind of odd, but I think we can leave it for now, since panel backgrounds and such might be revamped next.

Reply:
Ah, I think it's because of the disabled styling. 

My go-to place for checking it is the "settings" panel, which I think happens to have the same bg color as the "disabled" highlight.

I *believe* I've fixed it, but I can't figure out how to get to that screen.

Reply:
Info panel for a traffic signal > Delay tab
Really need to start that user guide / feature tour. ^^;

Reply:
Yeah, I dunno, I'd probably just be asking you what page of the user guide it's on. It's a big app!

Reply:
Confirmed fixed!

<img width="448" alt="Screen Shot 2021-01-21 at 3 26 10 PM" src="https://user-images.githubusercontent.com/217057/105414379-042f6300-5bfd-11eb-8b21-a744bb164523.png">

Reply:
(also fixed the overly dark checkbox for "Checkbox::custom"